dist/aw-parser.amd.js

```diff
@@ -165,15 +165,161 @@ var regex = {
     line_break: /^ {2}$/
 };
 
+var parser = {
```

Comment:
That makes the parsing stateful, but the state is not reset after parsing (e.g. scene_number keeps being incremented with each parsing).